src/pages/en/guides/troubleshooting.mdx

```diff
@@ -137,6 +137,10 @@ nodeLinker: "node-modules"
 
 In Astro, using a `<head>` tag works like any other HTML tag: it does not get moved to the top of the page or merged with the existing `<head>`. Because of this, you usually only want to include one `<head>` tag throughout a page. We recommend writing that single `<head>` and its contents in a [layout component](/en/core-concepts/layouts/).
 
+### An unwanted `<script>` or `<style>` is included
+
```

Comment:
```suggestion
When working with [conditionally displayed](https://docs.astro.build/en/core-concepts/astro-components/#dynamic-html) components, you may notice a component's `<script>` or `<style>` tags being included even if that component doesn't appear in the final output.
```

Adding a sentence to help describe the error before explaining it

Reply:
We use the both the term ["conditionally displayed"](https://docs.astro.build/en/core-concepts/astro-components/#dynamic-html) and ["conditionally rendered"](https://docs.astro.build/en/tutorial/2-pages/3/#conditionally-render-elements) elsewhere in the docs, so I chose the term that matches where I wanted to link!

Reply:
This might make it sound like the problem occurs *only* when conditionally rendered (and not, for example, just any unused import, like something you never bothered to remove when you decided not to render the component.)

I might choose something like:

> You may notice an imported component's `<script>` or `<style>` tags included in your HTML source even if that component doesn't appear in the final output. For example, this will occur with [conditionally rendered](https://docs.astro.build/en/core-concepts/astro-components/#dynamic-html) components that are not displayed.

Reply:
See my earlier comment - I don’t think the problem happens with any unused import, you have to use it in some way to trigger the issue!

Reply:
Has this been fixed then, because that for sure *used* to be the case??